JDI.Light/JDI.Light.Tests/UIObjects/Pages/Html5Page.cs

```diff
@@ -2,11 +2,18 @@
 using JDI.Light.Elements.Common;
 using JDI.Light.Elements.Composite;
 using JDI.Light.Interfaces.Common;
+using JDI.Light.Tests.UIObjects.Forms;
 
-namespace JDI.Light.Tests.UIObjects.Pages
+namespace JDI.Light.Tests.UIObjects
 {
     public class Html5Page : WebPage
     {
+        [FindBy(Css = "#avatar")]
+        public FileInput FileInput;
+
+        [FindBy(XPath = "//a[@href='/JDI/images/jdi-logo.jpg']")]
+        public Link FileDownload;
```

Comment:
this can be a property instead of a field

Reply:
Other members are fields. Hwy do we need prop in this case? What should me written in setter/getter?